test/system/answers_test.rb

```diff
@@ -81,15 +81,15 @@ class AnswersTest < ApplicationSystemTestCase
     end
 
     # Watcherに通知される
-    visit_with_auth '/notifications/unread', 'kimura'
+    visit_with_auth '/notifications?query=unread', 'kimura'
```

Comment:
`query` -> `status` にしたので `status` っぽそう？ テストが通っているのは `query=unread` だとこのパラメータの処理は controller にないので、単純にすべての通知が出るので assertion にも引っかからないって感じですかね。それ自体は挙動として特に問題ないような気がするので、`status=unread` にしたときにテストがちゃんと通っていれば OK な気がします。

```suggestion
    visit_with_auth '/notifications?status=unread', 'kimura'
```

Reply:
https://github.com/fjordllc/bootcamp/pull/3110/files#r698380992 にも記載しましたが、提示していただいたコードで解決しました！